### ISSUE TITLE
Batch Normalization freeze

### ISSUE THREAD

**Issue by jizongFox** [kazuto1011/deeplab-pytorch#48]
Hi @kazuto1011 , thanks for this wonderful work. While reading your code, I noticed that before the training, you freezed the batch_normalization layers. 
##
model.train()
model.module.scale.freeze_bn()
##
I am wondering what is the propose of this freeze, and if i train a network from scratch, does it still make sense to impose the freeze_bn()?

Thanks by advance for your kind reply!

**Comment by kazuto1011** [comment]
If you start from scratch, you must train all the batch norm layers, with adequate batch size to estimate the statistics of the mini-batch. The reason to freeze the batch norm layers in training is to reduce the computational cost, but note that the layers are pre-trained on ImageNet/COCO. It is a common technique in training models that need a very large memory, like in semantic segmentation.

If you'd like to train the batch norm params with multiple GPUs, you cannot use this repository as is. Please search for "sync batch norm" and replace with it.
